pkgs/by-name/er/ersatztv/package.nix

```diff
@@ -0,0 +1,57 @@
+{
+  lib,
+  fetchFromGitHub,
+  dotnetCorePackages,
+  buildDotnetModule,
+  sqlite,
+  ffmpeg,
+  which,
+}:
+
+buildDotnetModule rec {
+  pname = "ersatztv";
+  version = "0.8.7-beta";
+
+  src = fetchFromGitHub {
+    owner = "ErsatzTV";
+    repo = "ErsatzTV";
+    rev = "v${version}";
+    sha256 = "sha256-G3yw9aYZABEyghCddpAoO71wrjVbWiVJALEMZSbr2Lg=";
+  };
+
+  propagatedBuildInputs = [ sqlite ];
```

Comment:
Why propagate?

Reply:
I'm not sure. I based this off the jellyfin package, and I know ErsatzTV uses SQLite too, so I left it in. I'm not super familiar with building in Nix, what exactly does propagating it do?

Reply:
_Disclaimer: I am probably wrong as I'm still confused by propagation too and how to use it_

Propagating the build input means that if other packages depend on this `ersatztv` one, they will be built as if they included `sqlite` in their build inputs. This is needed in several package ecosystem where dependencies need to be available at runtime.